config/settings.yml

```diff
@@ -261,3 +261,6 @@ users_deletable_by_admins:
   default: 0
 users_deletable_by_self:
   default: 0
+journal_aggregation_time:
+  default: 300
```

Comment:
I'd vote for a lower default here... Something like... 5 minutes?

Keep in mind that mails will be delayed by that amount of time, too...